### PR TITLE
CI/CD elements

### PR DESCRIPTION
**Pull request by funrun11**
* docker based package building
* docker image for visualization-api

---
This change is [Reviewable](https://reviewable.io/reviews/zaletniy/visualization-api/6)

### CODE REVIEW

**Comment by funrun11** [comment]
@oleksiishyman @kbhonagiri16

File: tools/build/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Dockerfile
+FROM ubuntu:xenial
```
Comment (kbhonagiri16) [inline tools/build/Dockerfile:2]: Should we be using only xenial?just a question

File: tools/build/build_deb.sh [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+rm -rf platform-visibility-all* || true
+fpm -s dir -t deb \
+  -n platform-visibility-all \
+  -v 1.0.0 \
+  --after-install /app/tools/build/debian/postinstall \
+  $PROJECT_ROOT/etc/platformvisibility/visualization-api/visualization-api.toml=/etc/platformvisibility/visualization-api/ \
+  $PROJECT_ROOT/build/linux-amd64/visualizationapi=/usr/bin/
+
+mkdir -p $ROJECT_ROOT/build/deb
```
Comment (oleksiishyman) [inline tools/build/build_deb.sh]: Typo

Comment (kbhonagiri16) [inline tools/build/build_deb.sh]: Project

File: tools/build/Dockerfile (hunk 1 above)
Comment (funrun11) [reply tools/build/Dockerfile:2]: Target OS os Ubuntu xenial. It is not an issue for container, but in order to minimize potential issues with package

**Comment by oleksiishyman** [comment]
works for me

**Pull request merged**